tableauserverclient/models/task_item.py

```diff
@@ -3,12 +3,18 @@
 
 
 class TaskItem(object):
-    def __init__(self, id_, task_type, priority, consecutive_failed_count=0, schedule_id=None, target=None):
+    def __init__(self, id_, task_type, priority, consecutive_failed_count=0, schedule_id=None,
+                 schedule_name=None, schedule_frequency=None, schedule_next_run_at=None,
+                 schedule_last_run_at=None, target=None):
         self.id = id_
         self.task_type = task_type
         self.priority = priority
         self.consecutive_failed_count = consecutive_failed_count
         self.schedule_id = schedule_id
+        self.schedule_name = schedule_name
```

Comment:
Since these are all schedule properties, could you have one schedule_item here instead? You can leave schedule_id there, and I can create an issue to deprecate that.

Reply:
Add schedule_item there, removed other schedule_... variables, except schedule_id. Thanks for the comment!